feature/gnoi/system/tests/copying_debug_files_test/copying_debug_files_test.go

```diff
@@ -25,8 +25,10 @@ import (
 	"github.com/openconfig/ondatra"
 )
 
-const (
-	process = "bgp"
+var (
+	bgpProcName = map[ondatra.Vendor]string{
+		ondatra.NOKIA: "sr_bgp_mgr",
```

Comment:
LGTM. one quick ask, can you help keep the existing "bgp" process from the author? You can put Arista for "bgp". Thanks!

Reply:
I'll add as I had originally suggested.